tests/tests/test-rpc/test-starknet-rpc.ts

```diff
@@ -80,4 +80,15 @@ describeDevMadara("Starknet RPC", (context) => {
 
     expect(contract_class).to.not.be.undefined;
   });
+
+  it("getTransactionByHash", async function () {
+    // Test bad hash.
+    const tx = await providerRPC.getTransactionByHash("abcd");
```

Comment:
I have an error here:

```
LibraryError: -32602: missing field `tx_hash` at line 1 column 27
```

I am not sure what I am missing into the `.ts` file.